Readme-osx.md

```diff
@@ -33,12 +34,16 @@ https://documentation.dnanexus.com/downloads
     sudo port install py27-pip py27-virtualenv
```

Comment:
I have not yet updated python to 3+ (although locally I run it with python39 without issues).